addon/components/o-s-s/input-group.ts

```diff
@@ -0,0 +1,23 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+
+interface OSSInputGroupArgs {
+  value?: string;
+  disabled?: boolean;
+  errorMessage?: string;
+  placeholder?: string;
+  prefix?: string;
+  suffix?: string;
```

Comment:
```suggestion
  prefix: string;
  suffix: string;
```
If it is mandatory, it should not be undefined right?